src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/glide/ShadowBitmapPool.java

```diff
@@ -0,0 +1,53 @@
+package com.microsoft.maui.glide;
+
+import android.content.ComponentCallbacks2;
+import android.content.Context;
+import android.content.res.Configuration;
+import android.graphics.Bitmap;
+import android.util.LruCache;
+import com.bumptech.glide.load.engine.bitmap_recycle.BitmapPool;
+import com.bumptech.glide.load.engine.bitmap_recycle.LruBitmapPool;
+import com.bumptech.glide.load.engine.cache.MemorySizeCalculator;
+
+public class ShadowBitmapPool extends LruBitmapPool implements ComponentCallbacks2 {
+    private static ShadowBitmapPool bitmapPool;
+
+    public static BitmapPool get(Context context) {
+        if (bitmapPool == null) {
+            synchronized (ShadowBitmapPool.class) {
+                if (bitmapPool == null) {
+                    // Use application context to avoid memory leaks
+                    Context applicationContext = context.getApplicationContext();
```

Comment:
Do you have any details why to use `getApplicationContext`?

It looks like the `ShadowBitmapPool` is used once per `PlatformWrapperView`, which would have the same lifetime as an activity anyway?

If we used a single `ShadowBitmapPool` for the entire application, seems like `getApplicationContext` would be appropriate in that case.

Reply:
See comment below: we use only one pool.